frontend/packages/dev-console/integration-tests/support/step-definitions/addFlow/create-from-devfile.ts

```diff
@@ -40,3 +40,11 @@ When('user clicks Create button on Devfile page', () => {
 When('user enters Name as {string} in DevFile page', (name: string) => {
   gitPage.enterWorkloadName(name);
 });
+
+And('user can see under Services section {string}', (msg: string) => {
+  cy.get('span').contains(msg);
```

Comment:
This is very generic, change this to:
```suggestion
  cy.get('[role="dialog"]').contains(msg);
```